src/main/scala/uk/gov/nationalarchives/pdi/test/WorkflowManager.scala

```diff
@@ -36,18 +36,37 @@ import scala.jdk.CollectionConverters._
   */
 object WorkflowManager {
 
+  /** Executes a Pentaho Kettle transformation with the option of parameters and plugins
+    *
+    * @param transformationIs the InputStream with the Kettle transformation
+    * @param workingDirectory the working directory for the transformation
+    * @param maybeParameters  an optional map of transformation parameters
+    * @param maybePlugins     an optional list of plugin classes to be used
+    * @return
+    */
+  @deprecated
+  def runTransformation(
+                         transformationIs: InputStream,
+                         workingDirectory: Path,
+                         maybeParameters: Option[Map[String, String]] = None,
+                         maybePlugins: Option[List[Class[_ <: StepMetaInterface]]] = None
```

Comment:
You don't want to add default arguments to this method or the compiler will get confused Only the new method should have default args.